signer/api/api.go

```diff
@@ -63,34 +64,36 @@ func KeyInfo(sigServices signer.SigningServiceIndex) http.Handler {
 				return
 			}
 		}
+		key := &pb.PublicKey{KeyInfo: &pb.KeyInfo{KeyID: &pb.KeyID{ID: tufKey.ID()}, Algorithm: &pb.Algorithm{Algorithm: tufKey.Algorithm().String()}}, PublicKey: tufKey.Public()}
```

Comment:
Switched to a multiline format here and in other places with similar statements. Good suggestion.